### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by konturn**
feat: install plugin for embeddable bulid status; remove periodic folder scan from TVM job; make it so that docker deploy job is done manually so we can do updates without rolling jenkins every time.

### CODE REVIEW

**Review by nibalizer — APPROVED** [review]

**Review by areusch — APPROVED** [review]

**Comment by areusch** [comment]
cc @leandron @mousius

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: deploy_jenkins
+on: 
+  workflow_dispatch: []
```
Comment (driazati) [inline .github/workflows/deploy.yml:3]: nit but it can just be blank
```suggestion
  workflow_dispatch:
```

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
```
Comment (driazati) [inline .github/workflows/deploy.yml:4]: also we should cancel previously running jobs automatically so if someone hits deploy twice in a row it doesn't mess things up:

```suggestion
concurrency:
  group: deploy_jenkins
  cancel-in-progress: true
```

**Review by driazati — APPROVED** [review]

**Review by areusch — APPROVED** [review]

**Comment by areusch** [comment]
thanks @konturn @driazati !

**Pull request merged**